app/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/views/Layout.scala.html

```diff
@@ -41,11 +42,22 @@
   hmrcLanguageSelectHelper: HmrcLanguageSelectHelper,
   link: Link
 )
-@(pageTitle: Option[String] = None, backLinkEnabled: Boolean = true, isPageFullWidth: Boolean = false)(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
+@(
+        pageTitle: Option[String] = None,
+        backLinkEnabled: Boolean = true,
+        isPageFullWidth: Boolean = false,
+        disableSessionExpired: Boolean = false
+)(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
 
 
 @headBlock = {
-    @Some(hmrcTimeoutDialogHelper(signOutUrl = signOutUrl))
+
+    @if(!disableSessionExpired){
+        @hmrcTimeoutDialogHelper(
+            signOutUrl = uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.routes.ServiceController.logout.url,
+            keepAliveUrl = Some(uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.routes.ServiceController.keepAlive.url)
+        )
+    }
     <link @CSPNonce.attr rel="stylesheet" href='@routes.Assets.versioned("../assets/stylesheets/print.css")' type="text/css">
 }
 
```

Comment:
Need to add the logic to render the NavBar HTML. The code below should replace the `@accountMenu`

```
    @{ request match {
        case req: RequestWithNavBar => req.navBar.getOrElse(Html(""))
        case _ => Html("")
    }}
```